File: problem2/regio14a.py
# デバッガ利用時など標準入力を使いたい場合は、変更を。
# ファイル名は環境によっては 'problem2/sample1.txt' などに変更を。
# input = open('sample1.txt', 'r').readline


def check(skip, n, vec):
    return True


def main():
    n = int(input())
    line = input()
    vec = list(map(int, line.split()))
    for skip in range(n + 1):
        if check(skip, n, vec):
            print(vec[skip])
            return


main()

File: problem2/regio14a.cpp
#include <iostream>
#include <vector>
using namespace std;

bool check(int skip, int n, vector<int> & vec) {
	// skip 番目を無視して、
	// 等差数列になっているかチェック
	return true; // or false
}

int main(void) {
	int n;
	cin >> n;
	vector<int> vec(n+1);
	for (int i = 0; i < n + 1;  i++) {
		cin >> vec[i];
	}
	for(int skip=0; skip<n+1; skip++) {
		if(check(skip, n, vec)) {
			cout << vec[skip] << endl;
			return 0;
		}
	}
	return 0;
}
